docs/sphinx/source/whatsnew/v0.9.0.rst

```diff
@@ -45,6 +45,10 @@ Breaking changes
   ``surface_azimuth`` are now required parameters for
   :py:func:`pvlib.modelchain.basic_chain` (:issue:`1028`, :pull:`1181`)
 
+* Removed the **kwargs parameters from :py:meth:`~pvlib.pvsystem.sapm` and
+  :py:meth:`~pvlib.pvsystem.calcparams_desoto` and 
+  :py:meth:`~pvlib.pvsystem.calcparams_cec` (:issue:`1118`, :pull:`1222`)
```

Comment:
Sorry I didn't catch this the first time... these links need to include the `PVSystem` class too e.g. 

```
:py:meth:`~pvlib.pvsystem.PVSystem.sapm`
```

Reply:
Also Sphinx is unhappy about the double asterisk in `**kwargs`:  `whatsnew/v0.9.0.rst:48: WARNING: Inline strong start-string without end-string.`

I suggest wrapping the **kwargs in backticks (like ``` ``**kwargs`` ```) to prevent sphinx from trying to parse it as RST.